### ISSUE TITLE
Please use WinGet Releaser

### ISSUE THREAD

**Issue by ppvnf** [ONLYOFFICE/DesktopEditors#1063]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my idea.

### Your idea.

ONLYOFFICE is already available on Winget (ONLYOFFICE.DesktopEditors) and it seems up to date, but would it be possible to use [WinGet Releaser](https://github.com/marketplace/actions/winget-releaser) to make future versions automatically available on Winget? 

(As of yet, Winget does not have the ability to carry out such automations natively: https://github.com/microsoft/winget-pkgs/issues/500)

Thanks to [vedantmgoyal2009](https://github.com/vedantmgoyal2009) efforts, ONLYOFFICE now is automatically updated on Winget, but this isn't ideal because:

- Sometimes automation may go offline for maintenance, hence it is better to use GitHub Action. It is also easier to maintain and implement new features in the action. All apps use the latest manifest schema and fields.

- Automation is generally preferred for non-GitHub packages (meaning: software which are hosted on GitHub) that cannot take advantage of the GitHub Action.

- Since all GitHub API requests are authenticated with my account, using action distrubutes the amount of traffic on GitHub. The action is triggered automatically on releases, so we don't need to fetch APIs or scrape webpages to check for updates periodically, and PRs are created with the software's developer account.

- Microsoft prefers publishers managing releases for their apps themselves instead of a third party. It isn't a hard-and-fast rule, note the word "prefers" here.

**Comment by ShockwaveNN** [comment]
Hi, I think this WinGet config is done by third party and not our official distribution method (correct me if I wrong) so better to contact the person who did it

**Comment by ppvnf** [comment]
That's correct. [vedantmgoyal2009](https://github.com/vedantmgoyal2009) is running a script that automatically updates ONLYOFFICE and several other apps. However, I am helping him ask publishers to use an automation script, such [WinGet Releaser](https://github.com/marketplace/actions/winget-releaser), so he doesn't need to run the script himself (see above the reasons why the current solution isn't ideal).

https://github.com/vedantmgoyal2009/vedantmgoyal2009/issues/931

**Comment by LuckySpecialist** [comment]
Hi @ppvnf!
Issued your request as Bug 59652.
Thanks for the feedback.

**Comment by Rita-Bubnova** [comment]
Sorry, we forgot to mention that the issue was fixed.
I close this issue. Feel free to comment or reopen it if you got further questions.
